orbit/constants/dlt.py

```diff
@@ -88,6 +88,16 @@ class RegressionSamplingParameters(Enum):
     REGRESSION_COEFFICIENTS = 'beta'
 
 
+class LatentSamplingParameters(Enum):
```

Comment:
Is this change meant to be part of this diff?

Reply:
I prefer labeling them in the same style as what we did for other parameters.  Otherwise, it would be a set of strings coming from nowhere from reader point of view.  I think this is more organized.